desktop-style.css

```diff
@@ -357,4 +357,33 @@
   .mobile {
     display: none;
   }
+
+  .popup-card {
+    padding: 2%;
+  }
+
+  .cloned-desk-img {
+    height: 40vh;
+    width: 80vw;
+  }
+
+  .pop-body-container {
+    display: flex;
+    flex-direction: row;
+    width: 85vw;
+    margin-top: 1vh;
+  }
+
+  .pop-info-container {
+    width: 35vw;
+  }
```

Comment:
@Reem-lab Hello, regarding the image, It is indeed different but in order to follow the Figma design and make the image display like that I would have to use the object-fit: cover; property on the image to display properly. You can actually see how this property was applied to the Image in Figma because of the way it is cropping out parts of the Image. While this option looks quite good in some displays, in others you can barely see the image because of the zoom this property has and in bigger displays, it is impossible to really see the most important elements of the image.
I know it is important to stick with the template as much as possible, but if this option harms the design is it necessary?
Do you still want me to apply this property? Is this the correct step or is there any other property or another way I can achieve a similar result without making the image look bad on certain displays? I already looked for other ways but they either distort the image completely or leave at least a little bit of white space so it is not distorted.